Harvest.Web/ClientApp/src/Expenses/UnbilledExpensesContainer.tsx

```diff
@@ -29,32 +40,36 @@ export const UnbilledExpensesContainer = () => {
         const expenses: Expense[] = await response.json();
 
         setExpenses(expenses);
+        setTotal(expenses.reduce((acc, cur) => acc + cur.total, 0));
       }
     };
 
     cb();
-  }, [projectId]);
+  }, [projectId, props.newExpenseCount]);
 
-  // This function closes the modal and deletes the entry from the db
-  const confirmModal = async (expenseId: number) => {
-    const response = await fetch(`/Expense/Delete?expenseId=${expenseId}`, {
+  const deleteExpense = async (expense: Expense) => {
+    if (! await confirmRemoveExpense()) {
```

Comment:
make sure to run prettier if you haven't, the space here after the bang strikes me as unformatted.